jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  coverageDirectory: '.coverage',
+  collectCoverageFrom: [
+    'src/**/*.{js,jsx}',
```

Comment:
at here can be just `.js` :thinking:

Reply:
please add the coverageThreshold:

```json
coverageThreshold: {
    global: {
      statements: 97,
      branches: 95,
      functions: 100,
      lines: 97
    },
  },
```

this will ensure that if any changes lower the coverage the test builder will trigger an error